Programming Languages/C/readme.md

```diff
@@ -17,13 +17,13 @@ This section serves as a curated collection of valuable resources for learning a
   - [File Management](#file-management)
 - [Tools and IDEs](#tools-and-ides)
 - [Online Editors and Compilers](#online-editors-and-compilers)
-- [C Projects](#c-projects)
+- [Projects](#projects)
   - [Beginner Level Projects](#beginner-level-projects)
   - [Intermediate Level Projects](#intermediate-level-projects)
   - [Advance Level Projects](#advance-level-projects)
 - [Youtube Channels](#youtube-channels)
 - [Coding Platforms](#coding-platforms)
-- [C Programming Courses](#c-programming-courses)
+- [Courses](#courses)
```

Comment:
In this link, can you identify if free or not? same format

Reply:
Sure, will add that